examples/plugin/timer-plugin.cpp

```diff
@@ -0,0 +1,36 @@
+#include "RAJA/util/PluginStrategy.hpp"
+
+#include <iostream>
+#include <chrono>
+
+class TimerPlugin : public RAJA::util::PluginStrategy
+{
+public:
+  void preLaunch(RAJA::util::PluginContext RAJA_UNUSED_ARG(p))
+  {
+    start_time = std::chrono::steady_clock::now();
```

Comment:
@davidbeckingsale do we want to use the RAJA Timer utility here instead?

Reply:
We could do - this is just an example though.